src/Voron/Data/Tables/Table.cs

```diff
@@ -2442,7 +2443,8 @@ public TableReport GetReport(bool includeDetails, StorageReportGenerator generat
 
             report.AddData(ActiveDataSmallSection, includeDetails);
 
-            report.AddPreAllocatedBuffers(TablePageAllocator, includeDetails);
+            var allocator = new NewPageAllocator(_tx.LowLevelTransaction, _tableTree);
```

Comment:
Why are you creating a new one here? We now no longer create it by default, but there is an `EnsureTablePageAllocator` that you can call instear.

Reply:
Good catch, no longer needed. It was a left over.